### PR TITLE
Enabling redux devtools in production

### PR DESCRIPTION
**Pull request by olemartinorg**
## Description
This change enables redux devtools in the production build, which brings with a few hundred bytes extra code, ~~as well as a 7.1mb `altinn-app-frontend.js.map` and a 556kb `altinn-app-frontend.css.map` to be published on altinncdn.no.~~

Adding these to the production build gives us a few advantages when debugging and reproducing strange bugs:
- We can record the redux state history, saving them to a file, making it possible to save and replay the state history for race condition bugs
- ~~Thrown exceptions in our own code will include references to the actual source files, lines and code that caused the error (instead of line 1, column 489984)~~

## Related Issue(s)
- #392
- #452

## Verification
- [x] **Your** code builds clean without any errors or warnings
- [x] Manual testing done (required)
- ~~[ ] Relevant automated test added (if you find this hard, leave it and we'll help out)~~
- [x] All tests run green

## Documentation
- [ ] User documentation is updated with a separate linked PR in [altinn-studio-docs.](https://github.com/Altinn/altinn-studio-docs) (if applicable)

### CODE REVIEW

**Comment by olemartinorg** [comment]
I spoke with @lorang92 and @altinnadmin about this, and we decided against the 7mb source map this time around. Having it checked into the git repo for cdn, hosted on github spaces, etc, would quickly balloon into much more disk usage than just 7mb.

We also considered:
- Hosting the source maps elsewhere
- Only copying over the source map for the major version to CDN (skipping source maps inside every minor/patch folder)
- Gzipping (1.8mb) or compressing with brotli (1.4mb), however it seems devtools are not ready for a `.js.map.gz` or `.js.map.br` yet - it didn't load correctly (might have been an issue with the wrong HTTP response headers being generated by my static file web server).

TL;DR: Skipping source maps for now

File: src/altinn-app-frontend/src/store/index.ts [hunk 1]
```diff
@@ -22,7 +22,7 @@ export const setupStore = (preloadedState?: PreloadedState<RootState>) => {
 
   const innerStore = configureStore({
     reducer: reducers,
-    devTools: isDev,
+    devTools: true,
```
Comment (lorang92) [inline src/altinn-app-frontend/src/store/index.ts:25]: Just a note: this [defaults](https://redux-toolkit.js.org/api/configureStore#devtools) to `true`, so strictly not needed config. But better to be explicit I guess! 

Neat 🥇

**Review by lorang92 — APPROVED** [review]

File: src/altinn-app-frontend/src/store/index.ts (hunk 1 above)
Comment (olemartinorg) [reply src/altinn-app-frontend/src/store/index.ts:25]: Oh, didn't know that! But yeah, I don't mind that being explicit.

**Comment by olemartinorg** [comment]
@ivarne I agree they would be useful, especially for more experienced developers, but as of now they might be problematic to add to the CDN. As the CDN is backed by a git repo, every new file needs to be checked into git, grows the history, and might affect our ability to serve the CDN at all (github static pages have storage limitations). I'm hoping for this to easier in the future, possibly after moving the [CDN to azure](/Altinn/altinn-studio/issues/7566). Right now, the risks outweighs the benefits, IMO.
